### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by latenssi** [orbitdb/orbit-core#23]
After a quite comprehensive refactoring the documentation is out of sync.

**Comment by iohzrd** [comment]
Yeah. I just noticed this. Particularly, the get(channelName) method is still in the documentation but doesn't exist anymore. Is this method (or an equivalent) going to come back? I cant seem to get the list of messages from a channel other than by sending a message. I tried prodding around feed and feed.load() but to no avail.
